Makefile

```diff
@@ -2,7 +2,7 @@
 SHELL = /bin/bash
 
 # Authorino version
-VERSION = $(shell git rev-parse HEAD)
+VERSION ?= 0.0.0
```

Comment:
Is this being used in any case?

Reply:
Besides for semantic versioning, I suppose not.